test/e2e/apps/rc.go

```diff
@@ -182,9 +210,9 @@ var _ = SIGDescribe("ReplicationController", func() {
 		json.Unmarshal(rcStatusUjson, &rcStatus)
 		framework.ExpectEqual(rcStatus.Status.Replicas, testRcInitialReplicaCount, "ReplicationController ReplicaSet cound does not match initial Replica count")
 
-		rcScalePatchPayload, err := json.Marshal(autoscalingv1.Scale{
-			Spec: autoscalingv1.ScaleSpec{
-				Replicas: testRcMaxReplicaCount,
+		rcScalePatchPayload, err := json.Marshal(v1.ReplicationController{
```

Comment:
this change does not make sense... this is marshaling replication controller to JSON to send to an endpoint that expects autoscaling.Scale JSON... it coincidentally happens to match, but only by coincidence

Reply:
Reverted in 9660e480c76